src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    inverse[robot[key]] = key;
+  }
+
+  return inverse;
 }
```

Comment:
The function `inverseRobot` is doing more work than necessary. It first checks for duplicate values in the input object by comparing each value with every other value. This results in a time complexity of O(n^2), which is not efficient for large inputs. Then it creates the inverse object. These two steps can be combined into one, which would result in a time complexity of O(n).